pittAPI.py

```diff
@@ -428,3 +429,123 @@ def get_person(self, query, maxPeople=10):
+    def __init__(self):
+        pass
+
+    def get_dining_locations(self):
+        return self.get_dining_locations_by_status(status=None)
+    
+    def get_dining_locations_by_status(self, status=None):
+        # status can be nil, open, or closed
+        # None     - returns all dining locations
+        # "all"    - same as None (or anything else)
+        # "open"   - returns open dining locations
+        # "closed" - returns closed dining locations
+        
+        #sample_url = "https://m.pitt.edu/dining/index.json?_region=kgoui_Rcontent_I1_Ritems&_object_include_html=1&_object_js_config=1&_kgoui_page_state=eb95bc72eca310cbbe76a39964fc7143&_region_index_offset=15&feed=dining_locations&start=15"
+        # the _region_index_offset is optional
+        # -- seems like it's only for the id of the li for the html
+        dining_locations = {}
+        
+        end_loop = False
+        load_more = False
+        counter = 0
+        while( not end_loop ):
+            load_more = False
+            url = "https://m.pitt.edu/dining/index.json?_region=kgoui_Rcontent_I1_Ritems&_object_include_html=1&_object_js_config=1&_kgoui_page_state=eb95bc72eca310cbbe76a39964fc7143&feed=dining_locations&start=" + str(counter)
+            data = json.load(urlopen(url))
+            soup = BeautifulSoup(data['response']['html'], 'html.parser')
+            res = soup.find_all("div", class_="kgoui_list_item_textblock")
+
+            for i in res:
+                if( i.find("span").getText() != "Load more..." ):
+                    if( i.find("div") != None ):
+                        if( (('Next:'     in i.find("div").getText()) and status != "open"   ) or
+                            (('Next:' not in i.find("div").getText()) and status != "closed" ) ):
+                            #dining_locations.append([
+                            #    #str(i.find("span").getText()),
+                            #    #str(i.find("div").getText().replace(u'\u2013', '-').replace('\n', ''))
+                            #    i.find("span").getText(),
+                            #    i.find("div").getText().replace(u'\u2013', '-').replace('\n', '')
+                            #])
+                            dining_locations[self._encode_dining_location(i.find("span").getText())] = {
+                                "name"   : i.find("span").getText(),
+                                "hours"  : i.find("div").getText().replace(u'\u2013', '-').replace('\n', '').replace('Next: ', ''),
+                                "status" : "closed" if "Next:" in i.find("div").getText() else "open"
+                            }
+                            end_loop = True
+                    elif(status != "open"):
+                        #dining_locations.append([i.find("span").getText(), ""])
+                        dining_locations[self._encode_dining_location(i.find("span").getText())] = {
+                            "name"   : i.find("span").getText(),
```

Comment:
There should be no spaces between a key and it's corresponding `:` according to pep8. Get rid of all key-colon spaces across the code.